pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1012,6 +1012,11 @@ func (ctrl *PersistentVolumeController) bind(volume *v1.PersistentVolume, claim
 	}
 	claim = updatedClaim
 
+	if claim.Spec.DataSource != nil && !isVolumeDataPopulated(volume) {
+		errStr := fmt.Sprintf("error binding volume %q to claim %q: claim has data source, but data is not yet populated to volume", volume.Name, claimToClaimKey(claim))
+		glog.V(2).Infof(errStr)
```

Comment:
should be `V(3)`?


---
something else to note - i think the error bubbles up to the PVC bind() callers, yet Infof() is called both here and in the callers.
ideally the pattern for using glog should be calling Infof() once, but i see the same for other logic in this function...so this change is conformant.